tests/netstats_l2/Makefile

```
@@ -1,10 +1,8 @@
 include ../Makefile.tests_common
 
-BOARD_PROVIDES_NETIF := airfy-beacon fox iotlab-m3 mulle native nrf51dk nrf51dongle \
+BOARD_PROVIDES_NETIF ?= airfy-beacon fox iotlab-m3 mulle native nrf51dk nrf51dongle \
```

Maybe use the following:
```suggestion
BOARD_PROVIDES_NETIF += airfy-beacon fox iotlab-m3 mulle native nrf51dk nrf51dongle \
```

Not sure if it's useful or not (if not you can ignore).

---

It should not make any difference if you specify this variable explicitly for its use with a single board. BTW, `netstat_l2` could be used with ESP8266 boards in same way.